src/5-sitemap.ts

```diff
@@ -15,8 +15,15 @@ export function createSitemapItems() {
 
     const lastMod = entry.meta.uuid.currentRevision?.date?.split('T')[0]
 
+    let url = alias
+
+    if(entry.meta.uuid.__typename === 'Course' && entry.id.includes('#')){
+      // title should be the course title here, but fetching the courses content only for that is a lot of data
```

Comment:
is this comment still relevant, or: how to uuid.title differ from course title?

Reply:
Ah, typo. CoursePage title would be needed. Also the info if the course page is the first one.